assets/js/src/core/modules/notifications/send-notification/components/user-select/user-select.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * This source file is available under the terms of the
+ * Pimcore Open Core License (POCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (https://www.pimcore.com)
+ *  @license    Pimcore Open Core License (POCL)
+ */
+
+import { useUserGetCollectionQuery } from '@Pimcore/modules/user/user-api-slice.gen'
+import { Select } from 'antd'
```

Comment:
Can't we use our select for that? :)

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/1515/commits/3e7064f5c8cbfede28bb96a7452236c353d895f3